mitodl/mit-learn
# calculate min_score before score adjustments

### What are the relevant tickets?
closes https://github.com/mitodl/hq/issues/6020

### Description (What does it do?)
This pr adjusts the search query to take the minimum score cutoff based on the original scores calculated for a text query, not the scores adjusted for staleness and completeness


### How can this be tested?
On the main branch

1) Make a text search query and take a screen shot of the results
2) Log in as an admin. Set "Resource Score Staleness Penalty" to 0 and "Maximum Incompleteness Penalty" to 0 for the same query as 1. Take a screen shot of the results

Switch to this branch. Make the same query as above. You should have the same results in the front page as you see in the screen shot for 1) and the same counts for All resources, courses, programs and learning materials as you see in your screen shot for 2). Changing the "Resource Score Staleness Penalty" and "Maximum Incompleteness Penalty"  should change the result order but not the counts of available resources
